airmozilla/starred/static/starred/js/star_event.js

```diff
@@ -35,6 +35,7 @@ var Stars = (function() {
                 if (index > -1) {
                     $element.addClass('star-on');
                 }
+                Stars.setToolTip(element);
```

Comment:
To avoid an extra `$element = $(element)` inside setToolTip, change this line to `Stars.setToolTip($element)` and change `setToolTip` accordingly.